content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,107 @@
+​
+>"_One of our biggest problems is not knowing what Gems are given to us, when installing or upgrading the application._"&mdash;**O3DE Research Participant**
+​
+We found that it was difficult for new users to wrap their mind around the concept of Gems and their use within projects. With 80 Gems native to O3DE, the typical usage varies over two patterns—users either retain the default configuration throughout their development process or they enable all Gems to shorten build and compilation time. It was important to establish an information hierarchy, because of the Gems' low ==findability== and ==discoverability==.
+​
+Findability
+: The ease with which information can be found within a system.
+​
+Discoverability
+: The degree to which information can be found within a system.
+​
+The fundamental objective for the system's architecture was to allow users to be aware of where they are, what they found, what is around, and what to expect. As a result, we conducted a card sort study where participants were asked to arrange a stack of virtual cards into distinct categories based on their understanding of Gems. We identified the key category for the faceted navigation to be *Features*, which would classify Gems on the basis of their distinctive attributes and functionalities.
+​
+![Animated O3DE UX card sort GIF](/images/blog/bjds-posts/o3de-card-sort.gif)
+​
+With the research findings in mind, we established a set of design principles. Our goal was to identify a design solution that aligns with these principles while addressing the needs of all users.
+​
+**Discover, Find Search**
+Users must be able to explore the Gem Catalog efficiently. We improved the searchability of the catalog with filter, sort, and search functions. This allows users to make appropriate decisions for their project during Gem selection.
+​
+**Reinforce Clarity to Foster Learning**
+The design must give users constant feedback to help them recognise and recover from errors. By providing help and documentation at all steps, and by creating a clear informational hierarchy we can help reduce error and encourage user confidence.
+​
+**Autonomy to Encourage Contribution and Maintain Consistency and Standards**
+Users must be able to make their own decisions if they are to feel encouraged to contribute to the engine. We must ensure a design system that helps users to build consistent experiences with ease.
+​
+With the initial redesign, we continued to make gradual changes—most notably, the option to view the figurative "shopping cart" of Gems was introduced. Users could now see a list of all Gems they had added for activation. In addition, they could now view extensive information about Gems and were able to refine the list by the status of the Gem.
+​
+![Animated O3DE FTUE gem catalog iteration GIF](/images/blog/bjds-posts/o3de-gem-catalog-iteration.gif)
+​
+
+### What's Next?
+
+​
```

Comment:
Its interesting that the formatting looks right here in the code review, but looking at the netlify preview page it seems all merged together. @erickson-doug this section below looks like its supposed to be a few separate paragraphs, but the preview shows it all merged together.